cvat-ui/src/components/actions-menu/actions-menu.tsx

```diff
@@ -16,8 +16,8 @@ interface Props {
     taskMode: string;
     bugTracker: string;
 
-    loaders: string[];
-    dumpers: string[];
+    loaders: any[];
```

Comment:
Could you add an interface definition for loader and dumper?

Reply:
I don't know, can I? I'd just have used the class name, but for some reason they are never used here, so I just followed the style.

Reply:
@bsekachev, can we use more specific types here than any for objects?

Reply:
@zhiltsov-max 

On the one hand, of course, we can declare temporary interfaces in ``cvat-ui`` for ``loaders`` and ``dumpers`` and specify them here.
But on the other hand it is only temporary solution. In general case need to rewrite ``cvat-core`` on typescript or at least create typings file for the library. This is quite huge change.